kratos/python/add_communicator_to_python.cpp

```diff
@@ -94,18 +59,18 @@ void AddCommunicatorToPython(pybind11::module &m)
         .def("GlobalNumberOfConditions", &Communicator::GlobalNumberOfConditions)
         .def("GlobalNumberOfMasterSlaveConstraints", &Communicator::GlobalNumberOfMasterSlaveConstraints)
         .def("GetNumberOfColors", &Communicator::GetNumberOfColors)
-        .def("NeighbourIndices", NeighbourIndicesConst, py::return_value_policy::reference_internal)
+        .def("NeighbourIndices", [](Communicator& rCommunicator){return rCommunicator.NeighbourIndices();}, py::return_value_policy::reference_internal )
         .def("SynchronizeNodalSolutionStepsData", &Communicator::SynchronizeNodalSolutionStepsData)
         .def("SynchronizeNodalFlags", &Communicator::SynchronizeNodalFlags)
         .def("SynchronizeOrNodalFlags", &Communicator::SynchronizeOrNodalFlags)
         .def("SynchronizeAndNodalFlags", &Communicator::SynchronizeAndNodalFlags)
         .def("SynchronizeDofs", &Communicator::SynchronizeDofs)
-        .def("LocalMesh", CommunicatorGetLocalMesh, py::return_value_policy::reference_internal )
-        .def("LocalMesh", CommunicatorGetLocalMeshWithIndex, py::return_value_policy::reference_internal )
-        .def("GhostMesh", CommunicatorGetGhostMesh, py::return_value_policy::reference_internal )
-        .def("GhostMesh", CommunicatorGetGhostMeshWithIndex, py::return_value_policy::reference_internal )
-        .def("InterfaceMesh", CommunicatorGetInterfaceMesh, py::return_value_policy::reference_internal )
-        .def("InterfaceMesh", CommunicatorGetInterfaceMeshWithIndex, py::return_value_policy::reference_internal )
+        .def("LocalMesh", [](Communicator& rCommunicator){return rCommunicator.LocalMesh();}, py::return_value_policy::reference_internal )
```

Comment:
can you use `overload_cast` instead? lambdas were only a workaround before

Reply:
Done